Author: Pino Toscano <pino@debian.org>
Description: Fix build failure on GNU/Hurd.
 Since BSD is defined, make sure to include <paths.h> to get the definition
 of _PATH_UTMP.
Last-Update: 2012-06-11
Forwarded: no

--- a/kdm/kfrontend/genkdmconf.c
+++ b/kdm/kfrontend/genkdmconf.c
@@ -48,6 +48,8 @@ Foundation, Inc., 51 Franklin Street, Fi
 #ifdef BSD
 # ifdef BSD_UTMP
 #  include <utmp.h>
+# elif defined(__GNU__)
+#  include <paths.h>
 # endif
 # ifndef _PATH_UTMP
 #  if defined(__FreeBSD_version) && __FreeBSD_version >= 900007
